hack/prepare-minor-release.sh

```diff
@@ -51,7 +51,7 @@ sed -i "" "1r hack/changelog-template.txt" $DOCS_TO_UPDATE_FOLDER/releases.md
 sed -i "" -e "s/%%TITLE%%/## NGINX Ingress Controller $ic_version/g" -e "s/%%IC_VERSION%%/$ic_version/g" -e "s/%%HELM_CHART_VERSION%%/$helm_chart_version/g" $DOCS_TO_UPDATE_FOLDER/releases.md
 
 # update docs
-find $DOCS_TO_UPDATE_FOLDER -type f -name "*.md" -exec sed -i "" "s/v$prev_ic_version/v$ic_version/g" {} +
+find $DOCS_TO_UPDATE_FOLDER -type f -name "*.md" -exec sed -i "" "s/$prev_ic_version/$ic_version/g" {} +
```

Comment:
this will replace the values in the `docs-web/releases` for previous releases. A bug.